webservice/utils/get-data-from-form-section.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var assign = require('es5-ext/object/assign');
+
+module.exports = function getDataFromSection(dataForm, result, typeMapper) {
+
+	if (!result) result = {};
+
+	var ownerType = dataForm.propertyMaster;
```

Comment:
What you call `ownerType` is not really a type but a prototype. I think the name is misleading.